experiments/azure/cyclecloud/gpu/README.md

```diff
@@ -0,0 +1,213 @@
+cd configs/amg2023/
+for i in {1..5}; do sbatch --output=../../data/amg2023/%x-%j-iter-${i}.out --error=../../data/amg2023/%x-%j-iter-${i}.err slurm-amg-32n.sh; done
+cd ../../data/amg2023
+oras push ghcr.io/converged-computing/metrics-operator-experiments/performance:azure-cyclecloud-cpu-32node-amg2023 amg2023
+```
+
+
+#### Kripke
+
+```bash
+cd configs/kripke/
+for i in {1..5}; do sbatch --output=../../data/kripke/%x-%j-iter-${i}.out --error=../../data/kripke/%x-%j-iter-${i}.err slurm-kripke-32n.sh; done
+cd ../../data/kripke
+oras push ghcr.io/converged-computing/metrics-operator-experiments/performance:azure-cyclecloud-cpu-32node-kripke kripke
+```
+
+
+#### Laghos
+
+```bash
+cd configs/laghos/
+for i in {1..5}; do sbatch --output=../../data/laghos/%x-%j-iter-${i}.out --error=../../data/laghos/%x-%j-iter-${i}.err slurm-laghos-32n.sh; done
+cd ../../data/laghos
+oras push ghcr.io/converged-computing/metrics-operator-experiments/performance:azure-cyclecloud-cpu-32node-laghos laghos
+```
+
+#### LAMMPS
+
+```bash
+cd configs/lammps/
+for i in {1..5}; do sbatch --output=../../data/lammps/%x-%j-iter-${i}.out --error=../../data/lammps/%x-%j-iter-${i}.err slurm-lammps-32n.sh; done
+cd ../../data/lammps
+oras push ghcr.io/converged-computing/metrics-operator-experiments/performance:azure-cyclecloud-cpu-32node-lammps lammps
+```
+
+#### MiniFE
+
+```bash
+cd configs/minife/
+for i in {1..5}; do sbatch --output=../../data/minife/%x-%j-iter-${i}.out --error=../../data/minife/%x-%j-iter-${i}.err slurm-minife-32n.sh; done
+cd ../../data/minife
+oras push ghcr.io/converged-computing/metrics-operator-experiments/performance:azure-cyclecloud-cpu-32node-minife minife
+```
+
+Don't forget to save the MiniFE yaml output files that generate in the PWD.
+
+#### Mixbench
+
+```bash
+cd configs/mixbench/
+for i in {1..5}; do 
+  for node in $( sinfo -N -r -l | tail -n +3 | awk '{print $1}' ); do 
+    sbatch --nodelist=${node} --output=../../data/mixbench/${node}-%x-%j-iter-${i}.out --error=../../data/mixbench/%x-%j-iter-${i}.err slurm-mixbench-1n.sh
+  done
+done
+oras push ghcr.io/converged-computing/metrics-operator-experiments/performance:azure-cyclecloud-cpu-32node-mixbench mixbench
+```
+
+#### Mt-Gemm
+
+```bash
+cd configs/mt-gemm/
+for i in {1..5}; do sbatch --output=../../data/mt-gemm/%x-%j-iter-${i}.out --error=../../data/mt-gemm/%x-%j-iter-${i}.err slurm-mt-gemm-32n.sh; done
+cd ../../data/mt-gemm
+oras push ghcr.io/converged-computing/metrics-operator-experiments/performance:azure-cyclecloud-cpu-32node-mt-gemm mt-gemm
+```
+
+#### Nek5000
+
+```bash
+mkdir /shared/nekrs
+cd /shared/nekrs/
+oras pull ghcr.io/converged-computing/metric-nek5000:libfabric-cpu-data
+for i in {1..5}; do sbatch --output=../../data/nekrs/%x-%j-iter-${i}.out --error=../../data/nekrs/%x-%j-iter-${i}.err slurm-nekrs-32n.sh; done
+cd ../../data/nekrs
+oras push ghcr.io/converged-computing/metrics-operator-experiments/performance:azure-cyclecloud-cpu-32node-nekrs nekrs
```

Comment:
All of these URIs need to be changed to `azure-cyclecloud-gpu-32` (right now you will overwrite the CPU results)

Reply:
Good catch.